cmd/consumer/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/emmaLP/gs-software-onboarding/internal/config"
+	"github.com/emmaLP/gs-software-onboarding/internal/consumer"
+	"go.uber.org/zap"
+)
+
+func main() {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	logger, err := configureLogger()
+	if err != nil {
+		log.Fatal("Failed to configure the logger", err)
```

Comment:
errors should be lowercase

Reply:
Not necessarily.  According to this https://github.com/golang/go/wiki/CodeReviewComments#error-strings they can be capitalised